Common/AR/Ar.md

```diff
@@ -0,0 +1,340 @@
+    roll: 0.0
+}
+locationDataSource: null
+translationFactor: 1000.0
+```
+
+## AR frameworks configuration
+
+All the files necessary to build an ArcGIS application with AR support are provided in the AR toolkit.
+To deploy the application on Android and iOS devices, extra steps are needed to give the permission
+to the application.
+
+### Configuration for iOS devices
+
+1. Open the `Info.plist` file and add the following key at the end of the file, just before `</dict>`,
+to give the access to the camera to the `ARKit`.
+```
+<key>NSCameraUsageDescription</key>
+<string>Camera access is needed for AR testing</string>
+```
+
+See [NSCameraUsageDescription Apple's documentation](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription?language=objc) for details.
+
+2. The project is ready to be build and run in an iOS device. You need to verify the compatibility of your device
+with the `ARKit`.
+
+### Configuration for Android devices
+
+For details about `ARCore` configuration, see [ARCore documentation](https://developers.google.com/ar/develop/c/enable-arcore).
+
+1. In Qt Creator.
+
+1a. Open the `Projects` mode in the left toolbar and select an Android kit.
+
+1b. In the `Build Settings` then `Build Android APK`, clic on `Create template` button.
+An `AndroidManifest.xml` and `gradle` files are added to the project.
+
+2. Open the `AndroidManifest.xml` file using the plain text editor (right clic on the filename and select
+`Open With` then `Plain Text Editor`).
+
+2a. Under the line `<!-- Application arguments -->`, add the following line to enable the ARCore support.
+```
+<meta-data android:name="com.google.ar.core" android:value="required"/>
+```
+
+The value can be `optional` or `required`. An AR optional application can be installed and run on devices
+that don't support `ARCore` and the `Google Play Services for AR` is not automatically installed
+with the app. An AR required application is available only on devices that support `ARCore` and the
+`Google Play Services for AR` is automatically installed.
+
+For details, see [Enable ARCore documentation](https://developers.google.com/ar/develop/java/enable-arcore#manifest).
+
+2b. Under the line `<!-- %%INSERT_FEATURES -->`, add the following line to give the access to
+the camera to ARCore.
+```
+<uses-feature android:name="android.hardware.camera.ar" android:required="true"/>
+```
+
+2c. Set the minimal SDK version `android:minSdkVersion` to 24 for AR required application and 14 for
+AR optional application.
+```
+<uses-sdk android:minSdkVersion="24" android:targetSdkVersion="28"/>
+```
+
+3. Open the `build.gradle` file. In `dependencies`, add the ARCore dependency:
+```
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar', '*.aar'])
+    implementation 'com.google.ar:core:1.11.0'
+}
+```
+
+4. The project is ready to be built and run on an Android device. You need to verify the compatibility of your device
```

Comment:
Is there a link to any doc that explains which devices are compatible for Android?

Reply:
```

4. The project is ready to be build and run in a compatible Android device. ARCore requires Android 7.0 or later.
The list of compatible devices can be found in
[ARCore supported devices](https://developers.google.com/ar/discover/supported-devices)
```

Reply:
Should be built not build.